lib/item.rb

```diff
@@ -1,3 +1,10 @@
 class Item
 
+    attr_reader :name, :price
+
+    def initialize(item_details)
+        @name = item_details[:name]
+        @price = item_details[:price]
```

Comment:
The interaction pattern asks the price string to be cleaned (remove the $) and converted to a float.